deployments/bridges/rialto-millau/entrypoints/relay-messages-to-millau-generator-entrypoint.sh

```diff
@@ -3,29 +3,83 @@
 # THIS SCRIPT IS NOT INTENDED FOR USE IN PRODUCTION ENVIRONMENT
 #
 # This scripts periodically calls the Substrate relay binary to generate messages. These messages
-# are sent from the Millau network to the Rialto network.
+# are sent from the Rialto network to the Millau network.
 
 set -eu
 
 # Max delay before submitting transactions (s)
-MAX_SUBMIT_DELAY_S=${MSG_EXCHANGE_GEN_MAX_SUBMIT_DELAY_S:-60}
+MAX_SUBMIT_DELAY_S=${MSG_EXCHANGE_GEN_MAX_SUBMIT_DELAY_S:-30}
 MESSAGE_LANE=${MSG_EXCHANGE_GEN_LANE:-00000000}
-MESSAGE=${MSG_EXCHANGE_MESSAGE:-Remark}
+FERDIE_ADDR=5CiPPseXPECbkjWCa6MnjNokrgYjMqmKndv2rSnekmSK2DjL
 
-while true
-do
-	# Sleep a bit between messages
+SHARED_CMD="/home/user/substrate-relay submit-rialto-to-millau-message"
+SHARED_HOST="--rialto-host rialto-node-bob --rialto-port 9944"
+DAVE_SIGNER="--rialto-signer //Dave --millau-signer //Dave"
+ROOT_SIGNER="--rialto-signer //Alice --millau-signer //Alice"
```

Comment:
Alice also signs Rialto-PoA headers -> Rialto transactions. Using the same account here may lead to nonces conflict and delays (alerts) in Rialto-PoA -> Rialto headers sync. Possible solutions would be either to change account used by Rialto-PoA headers relay, or change Rialto root account.

I remember Tomek had doubts about keeping Root account origin - instead, for every 'root' action we'll probably need to use separate account. So alternate solution may be to remove root transactions at all. Otoh, the use-case for root account initially was to tune bridge parameters && we may have used something like `ensure_bridged_root(origin)?;` at the target chain. Not sure if we want to keep this use-case, or not? Probably it would be easier to submit target chain transactions directly (without going through bridge) and e.g. to use pallet owner for the same purposes. It is up to you

Reply:
Yeah, I'd skip root calls for now, I have some other idea that would replace them, cause the use case is really unclear.